x-pack/plugins/observability/public/components/app/section/apm/mock_data/apm.mock.ts

```diff
@@ -0,0 +1,171 @@
+/*
```

Comment:
Is this still needed?

Reply:
I moved all mocks to `pages/overview/` they will be used to show the storybook.